Payload/bleemsync/etc/bleemsync/FUNC/0050_bleemsync.funcs

```diff
@@ -136,45 +95,26 @@ execute_bleemsync_func(){
   start=$(date +%s%N)
 
   # Create gaadata on tmpfs
-  mkdir -p "/var/volatile/gaadatatmp/system/"
   if [ "$link_EMMC_and_USB" = "1" ]; then
-    mkdir -p "/var/volatile/gaadatatmp/databases/"
-    cp -f "$mountpoint/system/databases/regional.db" "/var/volatile/gaadatatmp/databases/"
+    mkdir -p "${VOL_GAADATA}/databases/"
+    cp -f "$mountpoint/system/databases/regional.db" "${VOL_GAADATA}/databases/"
   else
-    ln -s "$mountpoint/system/databases" "/var/volatile/gaadatatmp/"
+    ln -s "$mountpoint/system/databases" "${VOL_GAADATA}"
   fi
-  ln -s "$mountpoint/system/region" "/var/volatile/gaadatatmp/geninfo"
-  ln -s "$mountpoint/system/bios" "/var/volatile/gaadatatmp/system/bios"
-  ln -s "$mountpoint/system/preferences/system" "/var/volatile/gaadatatmp/preferences"
+  ln -s /gaadata/geninfo "${VOL_GAADATA}"
+  ln -s /gaadata/preferences "${VOL_GAADATA}"
+  ln -s /gaadata/system "${VOL_GAADATA}"
 
   # Create data on tmpfs
-  mkdir -p "/var/volatile/datatmp/sony/sgmo" "/var/volatile/datatmp/AppData/sony"
-  # ln -s "/tmp/diag" "/tmp/datatmp/sony/sgmo/diag"
-  ln -s "/dev/shm/power" "/var/volatile/datatmp/power"
-  ln -s "$mountpoint/system/ui" "/var/volatile/datatmp/sony/ui"
-  ln -s "$mountpoint/system/preferences/user" "/var/volatile/datatmp/AppData/sony/ui"
-  ln -s "$mountpoint/system/preferences/autodimmer" "/var/volatile/datatmp/AppData/sony/auto_dimmer"
-  cp -fr "/usr/sony/share/recovery/AppData/sony/pcsx" "/var/volatile/datatmp/AppData/sony/pcsx"
+  ln -s /gaadata/system/bios "${VOL_DATAPCSX}"
+  ln -s /usr/sony/bin/plugins "${VOL_DATAPCSX}"
 
-  PrevIFS="$IFS"
-  IFS=$'\t\r\n'
-  i=1
-  for game in `ls -d -v -1 $mountpoint/games/**`
-  do
-    if [ -d "${game}" ]; then
-      game_dir="${game}"
-      [ -d "${game_dir}/GameData" ] && game_dir="${game}/GameData"
-      ln -s "${game_dir}" /var/volatile/gaadatatmp/$i
-      rm -rf /var/volatile/datatmp/AppData/sony/pcsx/$i
-      ln -s "${game_dir}" /var/volatile/datatmp/AppData/sony/pcsx/$i
-      i=$((i+1))
-    fi
+  # Link games on /media
```

Comment:
In this process, both game data and the `.pcsx` user data are stored in the same directory.